README.md

```diff
@@ -1,21 +1,18 @@
-# Rossum SDK
+# Rossum API
```

Comment:
I think we could delete README.md and point to index.rst or rename it to README.rst.